README.md

```diff
@@ -1,26 +1,40 @@
 
 # Tutorials
 
+
 ## An introduction to the use of TERRA REF data and software
 
-Many of these tutorials have complex dependencies. In addition, many datasets are very large. These tutorials use the National Data Service Labs Workbench (NDS Labs Workbench). The NDS Labs Workbench provides the following:
+This repository provides a set of tutorials that are divided by data types and use cases. 
```

Comment:
Is there a good way / place to lay out the objectives of each tutorial, as opposed to just the broad subject areas? I get that they fall into a few categories, but knowing what I might doing with these tutorials will help motivate why I am stepping through the process.